content/en/docs/reference/config/analysis/ist0103/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: PodMissingProxy
+layout: analysis-message
+---
+
+This message occurs when the sidecar is not present or not working correctly.
+
+This most commonly occurs when you enable auto-injection but do not restart your
+pods afterwards, causing the sidecar to be missing.
+
+To resolve this problem, restart your pods and try again.
+
+For example, to restart the pods, use this command:
+
+{{< text bash >}}
+$ kubectl rollout restart deployment
```

Comment:
I don't think this is a valid command. Not sure what the right command is in this context, but checking kubectl's command-line help, I don't see rollout having a restart option.

Reply:
What version are you using? I ask because I see it here in v1.16.1 when I use kubectl rollout restart -h:
Restart a resource.
     Resource will be rollout restarted.

Examples:
  # Restart a deployment
  kubectl rollout restart deployment/nginx
  
  # Restart a daemonset
  kubectl rollout restart daemonset/abc

Reply:
I'm using 1.14 which is the latest version installed with the google gcloud stuff.

Reply:
@therealmitchconnors This is what we were discussing yesterday.

Reply:
Does anyone have the command to restart the pods for kubectl v 1.14? I can add it to the docs instead.

Reply:
`kubectl patch deployment  -p "{\"spec\":{\"template\":{\"metadata\":{\"labels\":{\"date\":\"`date +%s`\"}}}}}"` is one such command

There is a reason we use this one. There was actually a popular blog post recently criticizing our old docs for how to trigger a rollout as "look like a joke"